doc/developers-guide/mod_muc_light_developers_guide.md

```diff
@@ -27,16 +27,29 @@ All source files can be found in `apps/ejabberd/src`.
   Not recommended for production systems (less efficient than modern codec and requires more round-trips).
 
 * `mod_muc_light_codec_modern.erl`
-  An implementation of modern MUC protocol, described in the XEP. 
+  An implementation of modern MUC Light protocol, described in the XEP.
   Supports all MUC Light features.
 
+* `mod_muc_light_commands.erl`
+  MUC Light-related commands.
+  They are registered in `mongoose_commands` module, so they are available via REST API.
```

Comment:
They are registered in the `mongoose_commands` module, so they are available via the REST API.